<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Control">
      <attribute name="title">
       <string>打印机控制台</string>
      </attribute>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>421</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextBrowser
{
	
	color: rgb(255, 255, 255);
	
	background-color: rgb(0, 0, 0);
	font: 8pt &quot;微软雅黑&quot;;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>521</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>421</width>
         <height>281</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="outlong">
        <property name="geometry">
         <rect>
          <x>327</x>
          <y>168</y>
          <width>58</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ePlus">
        <property name="geometry">
         <rect>
          <x>176</x>
          <y>168</y>
          <width>68</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>挤出</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>98</x>
          <y>246</y>
          <width>219</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="excurdSpeed">
        <property name="geometry">
         <rect>
          <x>98</x>
          <y>209</y>
          <width>110</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>5000</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>82</x>
          <y>173</y>
          <width>42</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>设置回抽速度</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendGcode">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>246</y>
          <width>75</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>127</x>
          <y>174</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
       <widget class="QPushButton" name="eDele">
        <property name="geometry">
         <rect>
          <x>252</x>
          <y>168</y>
          <width>68</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>回抽</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>28</x>
          <y>173</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>243</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Gcode指令</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tempture">
        <property name="geometry">
         <rect>
          <x>294</x>
          <y>110</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Temp</string>
        </property>
       </widget>
       <widget class="QPushButton" name="yDele">
        <property name="geometry">
         <rect>
          <x>196</x>
          <y>67</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Y+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xHome">
        <property name="geometry">
         <rect>
          <x>44</x>
          <y>25</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>XHome</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zDele">
        <property name="geometry">
         <rect>
          <x>294</x>
          <y>68</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Z-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="yHome">
        <property name="geometry">
         <rect>
          <x>196</x>
          <y>25</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>YHome</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zPlus">
        <property name="geometry">
         <rect>
          <x>294</x>
          <y>26</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Z+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="yPlus">
        <property name="geometry">
         <rect>
          <x>44</x>
          <y>67</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Y-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xDele">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>109</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>X-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zHome">
        <property name="geometry">
         <rect>
          <x>196</x>
          <y>109</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ZHome</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xPlus">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>25</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>X+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xDele_2">
        <property name="geometry">
         <rect>
          <x>44</x>
          <y>109</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>打印机设置</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>预加热时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>39</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>40</y>
         <width>43</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>分钟</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>44</x>
         <y>73</y>
         <width>65</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>加热温度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>73</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>35</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>74</y>
         <width>43</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCustomPlot" name="cusplot" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>411</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>58</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>耗材半径</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="filamentDiameter">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>109</y>
         <width>112</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>2980</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>146</y>
         <width>70</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>挤出头宽度</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="extrusionWidth">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>147</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>4000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>43</x>
         <y>180</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>打印速度</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="printSpeed">
       <property name="geometry">
        <rect>
         <x>113</x>
         <y>180</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>./Plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
